embulk-output-mysql/src/main/java/org/embulk/output/mysql/MySQLOutputConnection.java

```diff
@@ -101,6 +103,52 @@ protected String buildCollectMergeSql(List<TableIdentifier> fromTables, JdbcSche
         return sb.toString();
     }
 
+    private String buildSwapTableSql(TableIdentifier fromTable, TableIdentifier toTable)
+    {
+        String suffix = "_embulk_swap_tmp";
+        String uniqueName = String.format("%016x", System.currentTimeMillis()) + suffix;
```

Comment:
Two points:

1. Do you have any more ideas that could be much better to guarantee the uniqueness?  If we could use `TEMPORARY TABLE` in this context, it could be much more robust, but we may not be able to use it here...
2. Can you leave a comment to explain "NOTE: The table name should be always shorter than 64 characters." with the link there?